frontend/src/pages/modelServing/customServingRuntimes/utils.ts

```diff
@@ -175,3 +188,28 @@ export const getKServeTemplates = (
     getTemplateEnabledForPlatform(template, ServingRuntimePlatform.SINGLE),
   );
 };
+
+export const servingRuntimeTemplate = ({
```

Comment:
Tiny nit: since this function triggers a state change can we name it starting with `set`? e.g. `setSelectedServingRuntimeTemplate`